### ISSUE TITLE
Not parsing telemetry arguments

### ISSUE THREAD

**Issue by miloskriz** [Cardinal-Cryptography/aleph-node-runner#14]
Hi!

**Goal**: to enable an aleph-node-runner container to connect to public telemetry at telemetry.polkadot.io.

**Procedure**: as per instructions, the environmental file at env/validator is appended with the following two lines:
```
TELEMETRY_ENABLED=true
CUSTOM_ARGS="--telemetry-url wss://telemetry.polkadot.io/submit/ 1"
```

**Results**: after executing `./run_node.sh --name <NAME>`, the docker container is deployed OK, but aborts and exits immediately, the `docker logs <NAME>` shows:
> error: Found argument '1' which wasn't expected, or isn't valid in this context
> 
> USAGE:
>     aleph-node [OPTIONS] [SUBCOMMAND]
> 
> For more information try --help

digging a bit more with `docker inspect <NAME>`, it is showing that the following variables are passed to the container:
> (...)
>                 "VALIDATOR=true",
>                 "WS_MAX_CONNECTIONS=100",
>                 **"TELEMETRY_ENABLED=true",
>                 "CUSTOM_ARGS=--telemetry-url wss://telemetry.polkadot.io/submit/ 1",**
>                 "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
> (...)

which look good at first sight, thus the reasons to believe the problem occurs further down the line, when these variables are parsed finally into the aleph-node image as flags for the binary.

Thanks for further investigating this!!

**Comment by miloskriz** [comment]
A quick update on this issue, when using the following entries in env/validator as advised:

```
TELEMETRY_ENABLED=true
CUSTOM_ARGS="--telemetry-url 'wss://telemetry.polkadot.io/submit/ 1'"
```

the same result of `docker logs` is obtained:

> error: Found argument '1' which wasn't expected, or isn't valid in this context
> 
> USAGE:
>     aleph-node [OPTIONS] [SUBCOMMAND]
> 
> For more information try --help

and the following variables are found with `docker inspect`:

>                 "TELEMETRY_ENABLED=true",
>                 "CUSTOM_ARGS=--telemetry-url wss://telemetry.polkadot.io/submit/ 1",

thus confirming that the behaviour is similar to original one posted above.

**Comment by piotrMocz** [comment]
Hey @miloskriz, just letting you know that I'm actively working to resolve your issue, however with little luck so far. Here's the gist of the problem:
those variables are processed in a few stages.
* First one is evaluating the `env/validator` file, because docker's `--env-file` option cannot do variable substitution. This results in a `env/validator.env` file.
* Next, this file is passed to docker through the `--env-file` option and here's where the problem happens: we want the arguments to the binary to be exactly `--telemetry-url 'wss://telemetry.polkadot.io/submit/ 1'`; however, when I try to either add more quotes by escaping them or escape the space, docker adds slashes of its own.
* The last part is the contents of `$CUSTOM_ARGS` being appended to the args of the binary (inside `docker-entrypoint.sh`).
Long story short, I still haven't found a combination of quotes and slashes that works for this use case, I'll keep trying :)
As an aside, I also tried passing the `CUSTOM_ARGS` variable directly to docker with the `-e` option: roughly the same results.
